static/app/views/starfish/components/starfishProjectSelector.tsx

```diff
@@ -0,0 +1,43 @@
+import {updateProjects} from 'sentry/actionCreators/pageFilters';
+import {CompactSelect} from 'sentry/components/compactSelect';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import {Project} from 'sentry/types';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import useProjects from 'sentry/utils/useProjects';
+import useRouter from 'sentry/utils/useRouter';
+
+const ALLOWED_PLATFORMS = new Set(['python']);
+
+export function StarfishProjectSelector() {
+  const {projects} = useProjects();
+  const {selection} = usePageFilters();
+  const router = useRouter();
+
+  const projectOptions = projects
+    .filter(project => ALLOWED_PLATFORMS.has(project.platform ?? '') && project.isMember)
```

Comment:
hmm instead of allowing all python projects, should we just maintain an allowlist of projects? right now, we have two and might potentially add another two (codecov, snuba) and that might be it for a bit?

Reply:
Sure, I can change that around! What's the other allowed project aside from `sentry`?